GoodMorning/GoodMorning/HomeScene/Views/PretendardLabel.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  PretendardLabel.swift
+//  GoodMorning
+//
+//  Created by Sunny on 2023/08/10.
+//
+
+import UIKit
+
+final class PretendardLabel: UILabel {
+
+    init(text: String,
+         color: UIColor = .black,
+         alignment: NSTextAlignment = .center,
+         size: CGFloat,
+         weight: UIFont.Weight) {
+        super.init(frame: .zero)
```

Comment:
```suggestion
    init(
        text: String,
        color: UIColor = .black,
        alignment: NSTextAlignment = .center,
        size: CGFloat,
        weight: UIFont.Weight
    ) {
```

요런 식으로 개행 처리하면 좋을 거 같습니다!😁

Reply:
ㅠㅠ 왜 자꾸 컨벤션이 틀릴까요,...? 수정 완료했습니다!

> 커밋: 226a48f42b46c3deaf291074bda0516101d940e7